pypeit/biasframe.py

```diff
@@ -29,33 +27,27 @@ class BiasFrame(processimages.ProcessImages, masterframe.MasterFrame):
         # Child-specific Internals
         #    See ProcessImages for the rest
 
-    Parameters
-    ----------
-    file_list : list (optional)
-      List of filenames
-    spectrograph : str (optional)
-       Used to specify properties of the detector (for processing)
-       Attempt to set with settings['run']['spectrograph'] if not input
-    settings : dict (optional)
-      Settings for trace slits
-    master_key : str (optional)
-      Setup tag
-    det : int, optional
-      Detector index, starts at 1
-    par : ParSet
-      PypitPar['calibrations']['biasframe']
-    master_key
-    master_dir
-    mode
-
-    Attributes
-    ----------
-    frametype : str
-      Set to 'bias'
-
-    Inherited Attributes
-    --------------------
-    stack : ndarray
+    Args:
+        spectrograph : str (optional)
+           Used to specify properties of the detector (for processing)
+           Attempt to set with settings['run']['spectrograph'] if not input
+        files (list,optional): List of filenames to process
+            master_key (:obj:`str`, optional):
+                The string identifier for the instrument configuration.  See
+                :class:`pypeit.masterframe.MasterFrame`.
+        det (:obj:`int`, optional):
+            The 1-indexed detector number to process.
+        par (:class:`pypeit.par.pypeitpar.FrameGroupPar`):
+            The parameters used to type and process the arc frames.
+        master_key (:obj:`str`, optional):
+            The string identifier for the instrument configuration.  See
+            :class:`pypeit.masterframe.MasterFrame`.
+        master_dir (str, optional): Path to master frames
+        reuse_masters (bool, optional): Load from disk if possible
+
+    Attributes:
```

Comment:
Sphinx actually shows this without it having to be called out here.  News to me...

Reply:
got it